incubator/xray/templates/xray-setup-conf.yaml

```diff
@@ -0,0 +1,86 @@
+apiVersion: v1
+kind: ConfigMap
```

Comment:
You should make this a secret, since it contains passwords.

Reply:
Now it does not contain any password.